src/tests/functional/form_submission_tests.ts

```diff
@@ -331,13 +331,22 @@ export class FormSubmissionTests extends TurboDriveTestCase {
     this.assert.ok(fetchOptions.headers["Turbo-Frame"], "submits with the Turbo-Frame header")
   }
 
-  async "test link method form submission"() {
-    await this.clickSelector("#link-method-redirect")
```

Comment:
For the record, this test passed by mistake. As the link was not within a frame, it was not turned into a form, and the redirect endpoint supports get requests.